### DISCUSSION TITLE
[Expanding relations] How to auto expand all relations ?

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#1645] [Q&A] [answered]
Can I use `expand=*` or `expand=[.*]` to auto expand all relations?
Or do you have any plan for it?

**Comment by ganigeorgiev** [comment, marked as answer]
No, this syntax is not supported at the moment. You have to specify each relation that you want to expand.

Allowing `*` could be ambiguous since it is not clear whether it will target both direct and indirect relations and in my opinion it is not worth the necessary dev effort for now.

**Reply by hungcrush** [reply]
I got it
